ale_linters/ada/adals.vim

```diff
@@ -0,0 +1,16 @@
+" Author: Bartek Jasicki http://github.com/thindil
+" Description: Support for Ada Language Server
+
+call ale#Set('ada_adals_executable', 'ada_language_server')
```

Comment:
I was looking into adding the Ada language server a while back and totally forgot about it until now. In any case, I added a couple more variables that the user could set:
`ada_ls_project` and `ada_ls_encoding` which correspond to "ada.projectFile" and "ada.defaultCharset" as listed here https://github.com/AdaCore/ada_language_server/#how-to-use-the-vscode-extension

I could give you the diff if you're interested.

Reply:
That will be awesome :) Sure, please send me a diff, I will try add it as soon as possible.

Reply:
```suggestion
call ale#Set('ada_adals_executable', 'ada_language_server')
call ale#Set('ada_adals_project, 'default.gpr')
call ale#Set('ada_adals_encoding', 'utf-8')

function! ale_linters#ada#adals#GetInitializationOptions(buffer) abort
    return {
    \   'ada.projectFile': ale#Var(a:buffer, 'ada_adals_project'),
    \   'ada.defaultCharset': ale#Var(a:buffer, 'ada_adals_encoding')
    \}
endfunction
```